aerospike-core/src/policy/concurrency.rs

```diff
@@ -27,15 +27,4 @@ pub enum Concurrency {
     /// extremely large batch sizes because each node can process the request immediately. The
     /// downside is extra threads will need to be created (or takedn from a thread pool).
     Parallel,
-
-    /// Issue up to N commands in parallel threads. When a request completes, a new request
```

Comment:
I just took the liberty to remove this completely.  @jonas32 's async patch offloads the responsibility for this stuff onto the runtime. Doing it allowed the other two to improve.